src/main/webapp/app/shared/components/customTab/custom-tab.tsx

```diff
@@ -73,14 +74,30 @@ export default connect(mapStateToProps)(CustomTab);
 interface ITabItemProps {
   currentTab: string;
   tabInfo: ITabInfo;
+  tabOnClick?: (tabName: string) => void;
 }
 
-const TabItem: React.FC<ITabItemProps> = ({ currentTab, tabInfo }) => {
+const TabItem: React.FC<ITabItemProps> = ({ currentTab, tabInfo, tabOnClick }) => {
   const isCurrentTab: boolean = tabInfo.tabName === currentTab;
   const btnClassName = classnames('tab-item', isCurrentTab ? 'active-tab' : '');
+
+  const handleClick = (tabName: string) => (event: any) => {
+    if (tabOnClick !== undefined) {
+      tabOnClick(tabName);
+    }
+  };
```

Comment:
no need to pass `tabName` as param, use closure instead.
```typescript
function handleClick(): void {
    if (!tabOnClick) return;
    tabOnClick(tabName); // tabName is gotten from closure, not from function param
}
```

Reply:
resolved in 9a53e28c330eb98c3e37e2eadf5e2a10eb26abe1